pgrx-pg-config/src/lib.rs

```diff
@@ -270,7 +326,12 @@ impl PgConfig {
 
     pub fn postmaster_path(&self) -> eyre::Result<PathBuf> {
         let mut path = self.bin_dir()?;
-        path.push("postmaster");
+
+        if self.major_version()? > 15 {
+            path.push("postgres")
+        } else {
+            path.push("postmaster");
+        }
```

Comment:
I am curious if checking for postgres unconditionally would break anything, but perhaps it is best to satisfy that curiosity later.